src/vs/workbench/services/notification/common/notificationService.ts

```diff
@@ -24,6 +25,10 @@ export class NotificationService extends Disposable implements INotificationServ
 		super();
 	}
 
+	getNotifications(): INotificationItem[] {
```

Comment:
planning to remove this and will use registry to directly get the keys instead from storage